sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -0,0 +1,157 @@
+# Azure Schema Registry client library for .NET
+
+The Azure Schema Registry service allows developers to provide and retrieve data schemas from a centralized repository for use in messaging systems.
+
+## Getting started
+
+### Install the package
+
+Install the Azure Schema Registry client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.Data.SchemaRegistry
```

Comment:
We generally use the dotnet cli for this snippet:
https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/servicebus/Azure.Messaging.ServiceBus#install-the-package

Also, you will need to specify the version since it is a preview.

Reply:
I based it off the AppConfiguration readme: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/appconfiguration/Azure.Data.AppConfiguration#install-the-package
I've updated it to use the dotnet CLI command.